maintenance/projects/js-toolkit/packages/liferay-npm-build-support/src/resources/start/index.js.ejs

```diff
@@ -1,3 +1,16 @@
+window.Liferay = {
+	Language: {
+		get: function(key) {
+			return key;
+		}
+	},
+	ThemeDisplay: {
+		getPathContext: function() {
+			return '';
```

Comment:
Feels like maybe at some point we could do better here... knowing which server we're running, maybe getting a path wouldn't be so hard and would make this work a bit better. Just thinking out loud here.

Reply:
:thinking: We are planning to let the `npm run start` functionality be managed by users, so there's no point in making it smarter, I think... 

Since this type of configurations will be done by hand, by a user knowing the exact nuances of his setup, it will be easier to mock `Liferay` as needed without having to contribute a new PR and wait for a new release of the JS Toolkit.

Reply:
Sounds good to me.. just pinging @dsanz here since he might be bored and wanting to add more stuff onto his thinking list (just kidding) 😂

Reply:
In fact, I'm preparing a PR to externalize the `npm run start` thing. I'll send it in minutes, and we can discuss there...